addons/binding/org.openhab.binding.elkm1/META-INF/MANIFEST.MF

```diff
@@ -0,0 +1,31 @@
+Manifest-Version: 1.0
+Automatic-Module-Name: org.openhab.binding.elkm1
+Bundle-ManifestVersion: 2
+Bundle-Name: Elk M1 Binding
+Bundle-SymbolicName: org.openhab.binding.elkm1;singleton:=true
+Bundle-Vendor: openHAB
+Bundle-Version: 2.5.0.qualifier
+Bundle-RequiredExecutionEnvironment: JavaSE-1.8
+Bundle-ClassPath: .
+Import-Package: 
+ com.google.common.collect,
+ org.eclipse.jdt.annotation;resolution:=optional,
+ org.eclipse.smarthome.config.core,
+ org.eclipse.smarthome.config.discovery,
+ org.eclipse.smarthome.core.library.types,
+ org.eclipse.smarthome.core.thing,
+ org.eclipse.smarthome.core.thing.binding,
+ org.eclipse.smarthome.core.thing.binding.builder,
+ org.eclipse.smarthome.core.thing.type,
+ org.eclipse.smarthome.core.types,
+ org.eclipse.smarthome.model.script.engine.action,
+ org.openhab.binding.elkm1,
+ org.openhab.binding.elkm1.handler,
```

Comment:
Please remove:

```
 org.openhab.binding.elkm1,
 org.openhab.binding.elkm1.handler,
```

and put all classes in `org.openhab.binding.elkm1.internal` or sub packages.